ui/src/app/machines/views/MachineDetails/MachineLogs/DownloadMenu/DownloadMenu.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { mount } from "enzyme";
+import { Provider } from "react-redux";
+import { MemoryRouter } from "react-router-dom";
+import configureStore from "redux-mock-store";
+
+import DownloadMenu from "./DownloadMenu";
+
+import type { RootState } from "app/store/root/types";
+import { rootState as rootStateFactory } from "testing/factories";
+
+const mockStore = configureStore();
+
+describe("DownloadMenu", () => {
+  let state: RootState;
+
+  beforeEach(() => {
+    state = rootStateFactory();
+  });
+
+  it("renders", () => {
+    state.machine.selected = [];
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter
+          initialEntries={[{ pathname: "/machines", key: "testKey" }]}
+        >
+          <DownloadMenu />
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("DownloadMenu").exists()).toBe(true);
```

Comment:
This isn't really testing anything other than the lines above. Should you look for the data-test?

Reply:
This is really just a placeholder as there's no functionality yet, I'll replace this with proper tests in some followups.